awx/main/models/jobs.py

```diff
@@ -1213,6 +1213,9 @@ class Meta:
 
     extra_vars_dict = VarsDictProperty('extra_vars', True)
 
+    def _set_default_dependencies_processed(self):
+        self.dependencies_processed = True
```

Comment:
this can't be right

Reply:
for SystemJob?

Reply:
think I'm on same page as seth

Reply:
oh sorry, so the `pass` is what gets used as the default for the normal type jobs. In other words, it uses the normal default, because the method doesn't do anything.